pre-reg-113-115/README.md

```diff
@@ -86,5 +86,11 @@ destinationObjectStore.s3.secret-key={destination object store secretkey}
 
 ### ArchivalScript.sql
 
+## ArchivalScript.sql 
```

Comment:
Can you rename this heading to move records to consumed tables and move this section to be top since this will be done first then then only the reencryption utility run will be done for remaining records.

Reply:
Done.